korangar/src/graphics/instruction.rs

```diff
@@ -177,8 +177,15 @@ pub struct EntityInstruction {
     pub world: Matrix4<f32>,
     pub texture_position: Vector2<f32>,
     pub texture_size: Vector2<f32>,
+    pub texture_top_left: Vector2<f32>,
```

Comment:
Why do we still need texture_position and texture_size? Can't we just create the vertex here? (Or even better: Just a simple 4x4 transformation matrix?)

Reply:
This function is only for calculating the vertex, I don't know if we can convert the vertices of texture to texture_position and texture_size.

Reply:
If you can't simplify it for now it's fine. We can refine this part another time.